src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,94 @@
+package duke.ui;
+
+import java.util.ArrayList;
+import duke.data.task.Task;
+
+
+public class Ui {
+    public Ui() {
+    }
+    private String dottedLine = "____________________________________________________________\n";
+
+    public void showLine() {
+        System.out.println(dottedLine);
+    }
+
+    public void greet() {
+        System.out.println(dottedLine +
+                "Hello! I'm Charlie\n" +
+                "What can I do for you?\n" +
+                dottedLine);
```

Comment:
Nice job following the 8-space indentation guideline for wrapped lines. However, there is another, intermediate rule that could apply:

![image](https://github.com/nus-cs2103-AY2324S1/ip/assets/44526554/4699997a-600e-4cf8-8faf-4abad22821fc)

```suggestion
        System.out.println(dottedLine
                + "Hello! I'm Charlie\n"
                + "What can I do for you?\n"
                + dottedLine);
```